lib/mongoid_session_store/store.rb

```diff
@@ -9,7 +9,7 @@ class Store < ActionDispatch::Session::AbstractStore
   private
 
     def generate_sid
-      BSON::ObjectId.new.to_s
+      SecureRandom.hex(32)
```

Comment:
Randomness means that it could happen the first week, or never. Would be nice to prevent it, since the consequences could be pretty bad.